src/com/sbezboro/standardplugin/listeners/PlayerChatListener.java

```diff
@@ -12,10 +12,23 @@
 
 public class PlayerChatListener extends EventListener implements Listener {
 
-	private final Pattern[] helpPatterns = new Pattern[] {
-			Pattern.compile(".*help.*", Pattern.CASE_INSENSITIVE),
-			Pattern.compile(".*how do i.*", Pattern.CASE_INSENSITIVE),
-			Pattern.compile(".*i['`\"]?m new.*", Pattern.CASE_INSENSITIVE),
+	private final Pattern[] genericHelpPatterns = new Pattern[] {
```

Comment:
Since we are getting into multiple types of help messages, it may be worthwhile converting this into a class-based system where the class defines a list of regex patterns and the resulting message that is sent to the player. Then in `onPlayerChat` below you can iterate through a list of 3 of these instances rather than having 3 separate `Pattern[]` constants